test/metabase/api/search_test.clj

```diff
@@ -679,38 +679,38 @@
   (testing "You should see Tables in the search results!\n"
     (mt/with-temp [Table _ {:name "RoundTable"}]
       (do-test-users [user [:crowberto :rasta]]
-        (is (= [(default-table-search-row "RoundTable")]
-               (search-request-data user :q "RoundTable"))))))
+                     (is (= [(default-table-search-row "RoundTable")]
```

Comment:
same here. `do-test-users` is a special macro. the original indentation was correct despite calva complaining so